stdlib/public/core/ManagedBuffer.swift

```diff
@@ -136,6 +136,38 @@ open class ManagedBuffer<Header, Element> {
   }
 }
 
+public extension ManagedBuffer {
+  @inline(never)
+  final class func reallocate<Buffer: ManagedBuffer>(
+    buffer: inout Buffer,
+    count: Int,
+    newMinimumCapacity: Int
+    ) {
+      precondition(_isBitwiseTakable(Header.self))
```

Comment:
are we all fine with just preconditioning all of those?

Reply:
I think that's fine. I'd like to also have a higher-level interface that falls back to manual copying for things that aren't bitwise takable, which should be doable if you require the count to be passed in like this.

Reply:
that sounds like the best idea to me, will add later